whois-scheduler/src/test/java/net/ripe/db/whois/scheduler/task/export/DecorationStrategyTest.java

```diff
@@ -51,17 +53,21 @@ public void decorate_dummify_allowed() {
 
     @Test
     public void decorate_dummify_not_allowed() {
+       final RpslObject personObject = RpslObject.parse("" +
+                "person: Ninja Person\n" +
+                "nic-hdl: NI124-RIPE\n");
+
         DecorationStrategy subject = new DecorationStrategy.DummifySplitFiles(dummifier);
-        Mockito.when(dummifier.isAllowed(3, object)).thenReturn(false);
```

Comment:
You have replaced the mntner `object` altogether so that case is no longer tested. Can you create a *new* test method just for person instead ?

Reply:
This test in wrongly written 
 Mockito.when(dummifier.isAllowed(3, object)).thenReturn(false);

This is never false if you pass a mntner...so I prefer to remove it by a person or a role without abuse mailbox

Reply:
I did add more accurate unit tests

Reply:
Thanks!